.github/workflows/extract-translations.yml

```diff
@@ -36,7 +36,12 @@ jobs:
           /bin/bash -eux ./scripts/extractCatalogue.sh
           rm catalog/autoupgrade/translations/ModulesAutoupgradeAdmin.en.xlf
           mv catalog/autoupgrade/translations/en-US/messages.en-US.xlf catalog/autoupgrade/translations/ModulesAutoupgradeAdmin.en.xlf
-          rmdir catalog/autoupgrade/translations/en-US
+          if [ -d "catalog/autoupgrade/translations/en-US" ] && [ "$(ls -A catalog/autoupgrade/translations/en-US)" ]; then
+            echo "::warning::The directory catalog/autoupgrade/translations/en-US is not empty. Potential loss of translations."
```

Comment:
In the current context, is there any other file we need to take in account?

Reply:
Seen privately: it would be better to keep failing at it means some translation domains are kept in the codebase.